tool/tsh/db.go

```diff
@@ -620,6 +628,17 @@ func prepareLocalProxyOptions(arg *localProxyConfig) (localProxyOpts, error) {
 		keyFile:   keyFile,
 	}
 
+	if arg.onNewConnection != nil {
+		dbCert, err := certFromPath(arg.profile.DatabaseCertPathForCluster(arg.cliConf.SiteName, arg.routeToDatabase.ServiceName))
```

Comment:
Does the cert can be reissued without restarting the local  proxy  ? If yes the dbCert will contains the stale cert data. Reading the cert from disk for  each connection is not effective but  maybe we can load cert periodically once per some timeout ?

Reply:
> Does the cert can be reissued without restarting the local proxy?

Yes, the cert can be reissued if the user runs other tsh commands (or if Teleport Connect tries to do something). However, the LocalProxy is also using the **stale** cert with today's implementation. 

Do we also want to add the functionality to overwrite the cert LocalProxy is using (without restarting) in this PR? There are many approaches we can do too.

Reply:
Can we cache the last cert loaded and check if it has expired? If it has expired then prompt user for credentials to reissue the cert.